CONTRIBUTING.md

```diff
@@ -321,12 +321,25 @@ See the [.travis.yml](.travis.yml) file for more details on how the tests are ru
 
 ## templatekit
 
-`templatekit` is a Python package, included in the templates repository, that provides helpers for rendering file and project templates (including custom Scons builders).
-`templatekit` also defines the dependencies needed to render the templates.
+`templatekit` is a Python package, included in the templates repository, that provides the infrastructure to render templates.
+Users generally use the `templatekit` command line app (implemented with [Click](http://click.pocoo.org/5/)).
+The Scons pipeline uses `templatekit`'s Python APIs directly.
+Behind the scenes, `templatekit` uses [Jinja](http://jinja.pocoo.org) and [Cookiecutter](https://cookiecutter.readthedocs.io) to render templates.
 
-To install `templatekit` for development, run `python setup.py develop` from the root of the repository.
+To install `templatekit` for development, run
+
+```bash
+pip install -e ".[dev]"
```

Comment:
Out of curiosity, what is `".[dev]"`?

Reply:
Installs extra development dependencies via `extras_require` in `setup.py` via setuptools.